woocommerce/woocommerce
# Added scroll to notices to cart page

Fixes #19685 

Testing instructions:
Before:
1. Add an item to cart
2. Go to cart, scroll to the bottom of the page
3. Add invalid coupon
4. Observe the notice is out of view
5. Add valid coupon
6. Observe the notice is out of view.
7. Remove coupon
8. Observe the notice is out of view.
9. Update quantity in cart
10. Observe the notice is out of view.


After:

1. Add an item to cart
2. Go to cart, scroll to the bottom of the page
3. Add invalid coupon
4. The view scrolls so that Notice is in view.
5. Add valid coupon
6. The view scrolls so that Notice is in view.
7. Remove coupon
8. The view scrolls so that Notice is in view.
9. Update quantity in cart
10. The view scrolls so that Notice is in view.
